package.json

```diff
@@ -70,21 +70,21 @@
     "es6-promise": "^3.0.2",
     "fast-isnumeric": "^1.1.1",
     "font-atlas-sdf": "^1.3.3",
-    "gl-cone3d": "^1.1.0",
+    "gl-cone3d": "git://github.com/archmoj/gl-cone3d.git#823cabfbb19d8b457730320010cb58df748ff9ee",
     "gl-contour2d": "^1.1.4",
-    "gl-error3d": "^1.0.7",
+    "gl-error3d": "git://github.com/archmoj/gl-error3d.git#d09c224fe25671fcd588a8c803fe83129156fbab",
     "gl-heatmap2d": "^1.0.4",
-    "gl-line3d": "^1.1.2",
+    "gl-line3d": "git://github.com/archmoj/gl-line3d.git#681398a5477f64cfd92ff7494215d5b6854e834b",
     "gl-mat4": "^1.2.0",
-    "gl-mesh3d": "^2.0.0",
+    "gl-mesh3d": "git://github.com/archmoj/gl-mesh3d.git#5828f6255d8c5360631d5feec65801bb104a88e8",
     "gl-plot2d": "^1.3.1",
-    "gl-plot3d": "^1.5.5",
+    "gl-plot3d": "git://github.com/archmoj/gl-plot3d.git#ce601a50cd3057899f6074f8ea5fcf0ab77f417c",
```

Comment:
https://github.com/gl-vis/gl-plot3d/compare/master...archmoj:rev-bounds

Reply:
There is also changes to gl-axes3d as a dependency of gl-plot3d here: https://github.com/gl-vis/gl-axes3d/compare/master...archmoj:rev-bounds